src/ray/raylet/node_manager.cc

```diff
@@ -576,9 +576,25 @@ void NodeManager::HandleReleaseUnusedBundles(
                        bundle_id.bundle_index()));
   }
 
-  // TODO(ffbin): Kill all workers that are currently associated with the unused bundles.
-  // At present, the worker does not have a bundle ID, so we cannot filter out the workers
-  // used by unused bundles. We will solve it in next pr.
+  // Kill all workers that are currently associated with the unused bundles.
+  for (const auto &worker_it : leased_workers_) {
```

Comment:
This part is copy-pasted. Can you consolidate this into a helper function?

Reply:
Code is not the same.